extensions/wikia/Recirculation/js/helpers/LateralHelper.js

```diff
@@ -0,0 +1,114 @@
+/*global define*/
+define('ext.wikia.recirculation.helpers.lateral', [
+	'jquery',
+	'wikia.window',
+	'wikia.abTest',
+], function ($, w, abTest) {
+	var libraryLoaded = false;
+
+	function loadLateral(callback) {
+		if (libraryLoaded && callback && typeof callback === 'function') {
```

Comment:
Checking for both truthiness of callback and its type is a bit redundant and silences "callback passed, but not a function" error. I would just check for truthiness and expect it to be a function if present.